packages/kit/src/core/prerender/prerender.js

```diff
@@ -259,10 +254,10 @@ export async function prerender({ config, entries, files, log }) {
 		for (const entry of config.prerender.entries) {
 			if (entry === '*') {
 				for (const entry of entries) {
-					enqueue(null, config.paths.base + entry); // TODO can we pre-normalize these?
```

Comment:
I don't understand what's going on with the removal of `config.paths.base`. Surely it needs to be there?

Reply:
also, not sure why you got rid of the comment. it's still a TODO

Reply:
I didn't understand the TODO. I thought it was referring to prepending the base

The base would not be needed as Vite's `baseMiddleware` strips out the `base` path from the URL as mentioned above https://github.com/sveltejs/kit/pull/5601#discussion_r924911591